templates/assessor_manager/_requests_list.html.twig

```diff
@@ -0,0 +1,57 @@
+{% import 'assessor_manager/_layout.html.twig' as macro %}
+
+{% for request in requests %}
+    <tr>
+        <td class="datagrid__table__col--hide-mobile">
+            {{ request.data.id }}
+        </td>
+        <td class="datagrid__table__col--left">
+            {{ request.data.firstName }} {{ request.data.lastName }}<br />
+            <span class="datagrid__table__col--hide-mobile">
+                <em>Lieu de vote :</em><br>
+                {{ request.data.assessorPostalCode }} {{ request.data.assessorCity }}
+            </span>
+            <br>
+            <span class="datagrid__table__col--hide-mobile">
+                <em>Lieu de résidence :</em><br>
+                {{ request.data.postalCode }} {{ request.data.city }}
+            </span>
+        </td>
+        <td>
+            {{ macro.displayVotePlaceWishes(request.data.votePlacesWishes) }}
+        </td>
+        <td>
+            {{ request.data.office }}
+        </td>
+        <td>
+            {% if not request.data.processed %}
+                <strong>En attente</strong>
+            {% else %}
+                Traitée
+                (le {{ request.data.processedAt|date('d/m/Y à H:i') }})
+            {% endif %}
+        </td>
+        <td>
+            {{ request.matchingVotePlacesCount }}
+        </td>
+        <td class="datagrid__table__col--hide-mobile">
+            {{ request.data.createdAt|date('d/m/Y H:i') }}
+        </td>
+        <td>
+            <a href="{{ path('app_assessor_manager_request', {'id': request.data.id}) }}"
+               id="request-link-{{ request.data.id }}"
+               target="_blank">
```

Comment:
do you really need target=blank for this link?

Reply:
I used the same behavior as procuration, I'll let PO decide :)

Reply:
👌